uber/site_sections/preregistration.py

```diff
@@ -411,7 +411,8 @@ def register_group_member(self, session, group_id, message='', **params):
             'group_id': group_id,
             'group': group,
             'attendee': attendee,
-            'affiliates': session.affiliates()
+            'affiliates': session.affiliates(),
+            'badge_cost': 0
```

Comment:
Even though group members technically have a badge_cost, we don't want it to show on the page. Their badge is (presumably) already paid for, so we don't want them to think they have to pay more on top of that if they're not kicking in extra.